spec/util/java_version_spec.rb

```diff
@@ -27,6 +27,10 @@
     expect(mod.bad_java_version?("1.8.0-beta")).to be_falsey
   end
 
+  it "should not mark non-standard javas as bad (IBM JDK)" do
+    expect(mod.bad_java_version?("pwi3270sr9fp10-20150708_01 (SR9 FP10)")).to be_falsey
```

Comment:
Ok!
   but I was more talking in case we know an "official" documentation for it :-) but no questions to the PR, this can go in for sure!